src/Activity/TrainWithExercises/index.js

```diff
@@ -48,10 +46,11 @@ class TrainWithExercise extends React.Component {
         />
       );
     }
-    let data = parallelogramData[this.state.index];
-    if (this.props.activityChosen === "mammals"){
-      data = mammalsData[this.state.index];
-    }
+    const data = {
+      mammals: mammalsData[this.state.index],
+      parallelogram: parallelogramData[this.state.index] 
+    }[this.props.activityChosen]
+    console.log('data ' + data.mammals);
```

Comment:
`data.mammals` will be undefined.

because it is not `data == { mammals: exampleM, parallelograms: exampleP } `

what happen is `data == { mammals: exampleM, parallelograms: exampleP }["mammals"] `
donc `data == exampleM`

The code as it is now should work, does it not?

Reply:
I don't find that `data` is undefined when I test it. It seem to work well